code42_connector.py

```diff
@@ -409,7 +413,8 @@ def _handle_remove_legal_hold_custodian(self, param, action_result):
         if legal_hold_membership_id is None:
             return action_result.set_status(
                 phantom.APP_ERROR,
-                f"Code42: User is not an active member of legal hold matter {matter_id} for action 'remove_legalhold_custodian'.",
+                f"Code42: User is not an active member of "
```

Comment:
Did `black` do this?

Reply:
No, but black is ok with it and I usually like to do this for longer strings.